src/sentry/issues/ingest.py

```diff
@@ -56,6 +57,13 @@ def save_issue_occurrence(
     group_info = save_issue_from_occurrence(occurrence, event, release)
```

Comment:
I think that ideally we'd skip calling this as well when the option isn't enabled. We need to do this to avoid calling https://github.com/getsentry/sentry/blob/2ec9adaa218e4097caa36e456c160b7cbe906b94/src/sentry/issues/ingest.py#L204 which will update counts on the issue and cause double counting.

I wonder if the best option here might be to add logic into `save_issue_from_occurrence` that just fetches the group and returns it when the option is disabled. If the group doesn't exist, then it just returns None so that we don't process it any further. 

I guess there should never be a case where the group doesn't exist since it'd have already been created by the transaction, but probably worth just being safe?

Reply:
that makes sense. I've updated `save_issue_from_occurrence`